src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>react-cart</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+    </head>
+    <body>
+        <div id="root">
```

Comment:
non appendere l'app react sul body, meglio usare un div che contiene tutto.
Questo perche spesso robe tipo font-google, o altri plugin, appendono sul body, e in quel caso react sbraga.
Se noi usiamo sempre un container siamo sicuri